pkgs/by-name/ok/oksh/package.nix

```
@@ -8,13 +8,13 @@
 
 stdenv.mkDerivation rec {
   pname = "oksh";
-  version = "7.5";
+  version = "7.6";
 
   src = fetchFromGitHub {
     owner = "ibara";
     repo = pname;
     rev = "${pname}-${version}";
-    sha256 = "sha256-r75jjH1E7/36b0ebjsZK7rAuOHEyaIta5Bi8nB4zVjo=";
+    sha256 = "sha256-iEV0ibEXwJioBaKN2Tuy0+SaVs8q0Ac4bImP8zhI7oI=";
```

```suggestion
    hash = "sha256-iEV0ibEXwJioBaKN2Tuy0+SaVs8q0Ac4bImP8zhI7oI=";
```
hash is preferred for sri notation
https://nixos.org/manual/nixpkgs/unstable/#fetchfromgithub

---

thanks, changed.